packages/vsphere-template-resolver/README.md

```diff
@@ -0,0 +1,27 @@
+# tkg-clusterclass-vsphere Package
```

Comment:
Did you mean "vsphere-template-resolver Package"?
Also, the text below still refers to tkg-clusterclass-vsphere.

Reply:
ahhh  my bad. corrected the typo